lib/platform/github/index.ts

```diff
@@ -783,20 +732,16 @@ export async function getIssueList() {
       config.issueList = await getRestIssues();
       return config.issueList;
     }
-    let [success, issues, cursor] = await getGraphqlIssues();
-    config.issueList = [];
-    while (success) {
-      for (const issue of issues) {
-        issue.state = issue.state.toLowerCase();
-        config.issueList.push(issue);
-      }
 
-      if (!cursor) {
-        break;
-      }
-      // istanbul ignore next
-      [success, issues, cursor] = await getGraphqlIssues(cursor);
-    }
+    const gql = new Graphql();
+    gql.setRepoOwner(config.repositoryOwner);
```

Comment:
Idea: use chainable setter functions (they return this), then the code looks much cleaner